### PR TITLE
test(sdk): teamsUserCredential integration test case

### PR DESCRIPTION
**Pull request by wenytang-ms**
* chore(sdk): get accesstoken from browser

* chore(sdk): fix error

* chore(sdk): remove .env file

* chore(sdk): explain the get sso token

### CODE REVIEW

File: packages/sdk/test/integration/scripts/StartSimpleAuth.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    process.env.IDENTIFIER_URI = `api://localhost/${process.env.CLIENT_ID}`;
+    process.env.OAUTH_AUTHORITY = `https://login.microsoftonline.com/${process.env.SDK_INTEGRATION_TEST_AAD_TENANT_ID}`;
+    process.env.AAD_METADATA_ADDRESS = `https://login.microsoftonline.com/${process.env.SDK_INTEGRATION_TEST_AAD_TENANT_ID}/v2.0/.well-known/openid-configuration`;
+    process.env.ALLOWED_APP_IDS = "1fec8e78-bce4-4aaf-ab1b-5451cc387264;5e3ce6c0-2b1f-4285-8d4b-75ee78787346;3d6e5c14-3406-4de2-8c00-e647aa15705a";
+    process.env.TAB_APP_ENDPOINT = "*";
+    process.env.SDK_INTEGRATION_TEST_USER_OBJECT_ID = process.env.SDK_INTEGRATION_TEST_USER_OBJECT_ID;
+}
+
+async function downloadSimpleAuth() {
+    const versionFilePath = path.join(__dirname, "../../../../fx-core/templates/plugins/resource/simpleauth/version.txt");
```
Comment (blackchoey) [inline packages/sdk/test/integration/scripts/StartSimpleAuth.js:20]: sdk may use different version than simple auth plugin, let's define a separate version here. Can be in the script.

Comment (wenytang-ms) [reply packages/sdk/test/integration/scripts/StartSimpleAuth.js:20]: sure

File: packages/sdk/test/integration/scripts/StartSimpleAuth.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    process.env.IDENTIFIER_URI = `api://localhost/${process.env.CLIENT_ID}`;
+    process.env.OAUTH_AUTHORITY = `https://login.microsoftonline.com/${process.env.SDK_INTEGRATION_TEST_AAD_TENANT_ID}`;
+    process.env.AAD_METADATA_ADDRESS = `https://login.microsoftonline.com/${process.env.SDK_INTEGRATION_TEST_AAD_TENANT_ID}/v2.0/.well-known/openid-configuration`;
+    process.env.ALLOWED_APP_IDS = "1fec8e78-bce4-4aaf-ab1b-5451cc387264;5e3ce6c0-2b1f-4285-8d4b-75ee78787346;3d6e5c14-3406-4de2-8c00-e647aa15705a";
+    process.env.TAB_APP_ENDPOINT = "*";
+    process.env.SDK_INTEGRATION_TEST_USER_OBJECT_ID = process.env.SDK_INTEGRATION_TEST_USER_OBJECT_ID;
+}
+
+async function downloadSimpleAuth() {
+    const versionFilePath = path.join(__dirname, "version.txt");
```
Comment (blackchoey) [inline packages/sdk/test/integration/scripts/StartSimpleAuth.js:20]: update the file name to reflect the version is simple auth version

**Review by blackchoey — APPROVED** [review]

File: packages/sdk/test/integration/scripts/StartSimpleAuth.js (hunk 2 above)
Comment (wenytang-ms) [reply packages/sdk/test/integration/scripts/StartSimpleAuth.js:20]: ok

**Pull request merged**